### ISSUE TITLE
Unread conversation count not updated when stream is reloaded

### ISSUE THREAD

**Issue by svbergerem** [diaspora/diaspora#6494]

**Comment by svbergerem** [comment]
For 0.5.4.0 there should be at least a temporary fix.

**Comment by denschub** [comment]
I marked this bug to block the upcoming release which I froze moments ago since I think this will make users angry. I'd be pretty happy to see a fix soon, if possible. :)

**Comment by AugierLe42e** [comment]
Notification count is, too, not refreshed anymore.

**Comment by svbergerem** [comment]
This issue still exists for the stream links in the left navbar.

**Comment by denschub** [comment]
True, thanks. Related to (and possibly in the future solved by) #6493.
